packages/data-service-generator/src/prepare-context.ts

```diff
@@ -68,7 +67,6 @@ export async function prepareContext(
   );
 
   context.plugins = plugins;
-  return [];
```

Comment:
why did you remove that?

Reply:
It is never called with the intent of creating a Module array